dashboard/app/views/levels/show.html.haml

```diff
@@ -79,8 +79,8 @@
 
 = render partial: 'levels/admin'
 
-- if @script && !@script.old_professional_learning_course? && !@level.is_a?(NetSim)
-  = render partial: 'levels/teacher_panel'
-
```

Comment:
- [ ] turn teacher panel back on - figure out what happens if you're a teacher with AI_TUTOR_ACCESS permission and both panels compete to show...?